experiments/optimal-core-allocation/README.md

```diff
@@ -60,28 +59,28 @@ core.cpu() # if run with USE snap collector
 
 #### How to run in distributed mode with custom configuration file
 
-This example shows how to run the experiment using configuration file.
-All experiment flags can be provided using configuration file. Command line flag `-foo_bar` is equal to `FOO_BAR` option in configuration file.
+This example shows how to run the experiment using a configuration file.
+All experiment flags can be provided using a configuration file. Command line flag `-foo_bar` is equal to `FOO_BAR` option in configuration file.
 
 1. Generate default configuration: 
 
 ```bash
 sudo optimal-core-allocation -config-dump >example-configuration.txt
```

Comment:
Our conf file acutally follows `ini` format, so I have started to use it in Sensitivity Profile Documentation (e.g. `config.ini`). This way I got syntax coloring :) What do you think about that?